content/blog/Installation.md

```diff
@@ -1,6 +1,113 @@
+When RHDH is already installed and used, it is required to avoid re-installing it by the Helm chart.
+There are few steps required add the Orchestrator to such environment:
+1. Install the backing services by the Helm chart by disabling the RHDH installation
+2. Manually patching the existing RHDH ConfigMap resources with the Orchestrator plugin configuration.
+3. Import the Orchestrator software templates into Backstage catalog
+
+To install the backing services without RHDH use the option `--set rhdhOperator.enabled=false`
+A complete command will look like:
+`helm upgrade -i --debug orchestrator orchestrator --set rhdhOperator.enabled=false` 
+Resulting in the installation of the Sonataflow Operator, Serverless Operators.
+Alternately, those operators can be installed directly from the operator catalog.
+
+In RHDH installation, there are two major ConfigMaps to edit, under the `backstage-system` or the `rhdh-operator` namespaces:
+`dynamic-plugins` - contains the configuration to enable and set the dynamic plugins.
+To add the orchestrator plugins, add into the `dynamic-plugins` configmap the following:
+```
+- disabled: false
+    package: "@janus-idp/backstage-plugin-orchestrator-backend-dynamic@1.5.3"
+    integrity: >-
+      sha512-l1MJIrZeXp9nOQpxFF5cw1ItOgA/p4xhGjKN12sg4Re8GC1qL+5hik+lA1BjMxAN6nKGWsLdFkgqLWa6jQuQFw==
+    pluginConfig:
+      orchestrator:
+        dataIndexService:
+          url: http://sonataflow-platform-data-index-service.sonataflow-infra
+        editor:
+          path: "https://sandbox.kie.org/swf-chrome-extension/0.32.0"
+  - disabled: false
+    package: "@janus-idp/backstage-plugin-orchestrator@1.7.8"
+    integrity: >-
+      sha512-wJtu4Vhx3qjEiTe/i0Js2Jc0nz8B3ZIImJdul02KcyKmXNSKm3/rEiWo6AKaXUk/giRYscZQ1jTqlw/nz7xqeQ==
+    pluginConfig:
+      dynamicPlugins:
+        frontend:
+          janus-idp.backstage-plugin-orchestrator:
+            appIcons:
+              - importName: OrchestratorIcon
+                module: OrchestratorPlugin
+                name: orchestratorIcon
+            dynamicRoutes:
+              - importName: OrchestratorPage
+                menuItem:
+                  icon: orchestratorIcon
+                  text: Orchestrator
+                module: OrchestratorPlugin
+                path: /orchestrator
+```
+The plugin versions may change and so their integrity values. To pick the latest, refer to the [helm chart](https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator/templates/rhdh-operator.yaml#L159). Make sure to set both the version and the integrity values.
+
+The `dataIndexService.url` should point to the service of the Data Index installed by the Chart/Operator.
+
+In the `app-config` ConfigMap add the following:
+```
+app:
+  backend:
+    csp:
+        frame-src:
+          - "https://sandbox.kie.org"
+      cors:
+        origin: {{ the URL for backstage}}
```

Comment:
is it correct?

Reply:
we cannot know in advance the value of this property. It depends on the target namespace, the Backstage CR name and hence - either the k8s service or the OCP Route CR with its host name.

Reply:
So what about adding info on how to set this value?

Reply:
that's a preexisting RHDH. the link should be known to the admin :-)